next.config.js

```diff
@@ -2,4 +2,7 @@ module.exports = {
   images: {
     domains: ['flathub.org', 'dl.flathub.org'],
   },
+  env: {
```

Comment:
Should this be here? Seems like something you used to test this locally?

Reply:
Yes, I will change it. Looks like there is new (maybe not THAT new, but I was not aware of it) way of adding environment variables to bundle, and not only to a next server. https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser